app/src/main/res/layout/dialog_claim_info.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <ImageView
+        android:layout_width="220dp"
+        android:layout_height="164dp"
```

Comment:
why so? 
Make it wrap content

Reply:
because this image is smaller then we need
=( it is from sell but we can not have duplicates